wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -9,576 +9,652 @@
+
+                {% dialog icon_name="globe" title="Dialog with success" id="dialog-3" subtitle="This is a testing subtitle" message_status="success" message_heading="Success! You've done the thing" %}
+                    <p class="w-base-text">This dialog message was generated by passing message_status=success as well as message_heading to the dialog template tag</p>
+                {% enddialog %}
             </div>
+            <div>
+                {% dialog_toggle classname='button button-primary' dialog_id='dialog-4' text='Dialog with warning' %}
 
-            <p>Tabs can also indicate errors:</p>
+                {% dialog icon_name="globe" title="Dialog with warning" id="dialog-4" subtitle="This is a testing subtitle" message_status="warning" message_heading="There was an issue with the thing" message_description="This is a subtext for the message" %}
+                    <p class="w-base-text">This dialog message was generated by passing message_status=warning as well as message_heading and message_description to the dialog template tag</p>
+                {% enddialog %}
+            </div>
+            <div>
+                {% dialog_toggle classname='button button-primary' dialog_id='dialog-5' text='Dialog with error' %}
 
-            <div class="w-tabs" data-tabs>
-                <div role="tablist" class="w-tabs__list">
-                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-errors-1' title='Tab 1' errors_count='5' %}
-                    {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-errors-2' title='Tab 2' errors_count='55' %}
-                </div>
+                {% dialog icon_name="globe" title="Dialog with critical error" id="dialog-5" subtitle="This is a testing subtitle" message_status="critical" message_heading="There was an issue with the thing" message_description="This is a subtext for the message" %}
+                    <p class="w-base-text">This dialog message was generated by passing message_status=critical as well as message_heading and message_description to the dialog template tag</p>
+                {% enddialog %}
             </div>
-        {% endpanel %}
+            <div>
+                {% dialog_toggle classname='button button-primary' dialog_id='dialog-6' text='Floating dialog' %}
 
-        {% panel id="progress" heading="Progress indicators" %}
-            <div id="progress-example" class="progress active">
-                <div class="bar">60%</div>
+                {% dialog theme="floating" icon_name="doc-full-inverse" id="dialog-6" title="Floating dialog" subtitle="This is a testing subtitle" %}
+                    <p class="w-base-text">This dialog allows other page content to be seen</p>
+                {% enddialog %}
             </div>
+        </div>
+    {% endpanel %}
+
+    {% panel id="header" heading="Header" %}
+
+        {% include "wagtailadmin/shared/header.html" with title="{{ self.page_title }}" %}
```

Comment:
```suggestion
        {% include "wagtailadmin/shared/header.html" with title=page_title %}
```

This is a Template include, so passing variables to it works a bit differently.

https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#include